guides/introduction/Embedded Schemas.md

```diff
@@ -92,57 +87,124 @@ end
 defmodule UserProfile do
   use Ecto.Schema
 
-  embedded_schema "profile" do
-    field :age, :integer
-    field :favorite_color, Ecto.Enum, values: [:red, :green, :blue, :pink, :black, :orange]
-    field :avatar_url, :string
+  embedded_schema do
+    field :online, :boolean
+    field :dark_mode, :boolean
+    field :visibility, Ecto.Enum, values: [:public, :private, :friends_only]
   end
 end
 ```
 
-Ta-da! Neat. Note we replaced the `embeds_one` macro by `embedded_schema`: `embeds_one` and `embeds_many` function like fields, similar to relations like `has_many`.
+It is important to remember that `embedded_schema` has many use cases independent of `embeds_one` and `embeds_many`. **Embedded schemas are schemas without persistence.** You can think of `embedded_schema` as something like a persistence agnostic schema. This makes embedded schemas ideal for scenarios where you want to manage structured data without necessarily persisting it. For example, when you want an intermediate data structure between your UI layer and your data layer, or when you want to independently manipulate embedded structs inside a parent schema.
```

Comment:
> For example... inside a parent schema, an `embedded_schema` is a good choice or something something.

Reply:
I feel like this is an awkward sounding paragraph. I tried to rework it inspired by your suggestion. Not exactly how you suggested. It's a really important point I think, for new ecto devs to learn, so I want it to be a good paragraph, but idk, writing's hard, lol. If you have more edits here from what I changed it to, would love to heard them :D